### PR TITLE
Spp 10706 winsorisation documentation

### PR DESCRIPTION
**Pull request by gibbardsteve**
Adding user notes for the Winsorisation method. Includes example data, walkthroughs and guides to use the method.

### CODE REVIEW

File: method-info/winsorisation/example-data/winsorisation_expansion_input.csv [hunk 1]
```diff
@@ -0,0 +1,11 @@
+reference,period,group,target_value,design_weight,l_value
+123456,202201,1001,50,100,3000
```
Comment (jhmjam) [inline method-info/winsorisation/example-data/winsorisation_expansion_input.csv:2]: quick question where did you get this data pr was it something you gathered/computed from the method code?

File: method-info/winsorisation/winsorisation.md [hunk 2]
```diff
@@ -0,0 +1,423 @@
...
+
+## Finding and Installing the Method
+
+You can find instructions on downloading and installing the method in the [Help centre](https://statisticalmethodslibrary.ons.gov.uk/help-centre/index) of the [ONS Statistical Methods Library](https://statisticalmethodslibrary.ons.gov.uk)
+
+## Using the Method
+
+### Overview
+
+The Winsorisation method is available as part of the statistical-methods-library package. It takes a dataframe as input that contains the data to be considered for Wisorisation. This method implements one-sided Winsorisation only.
```
Comment (jhmjam) [inline method-info/winsorisation/winsorisation.md:11]: * for Winsorisation

File: method-info/winsorisation/winsorisation.md [hunk 3]
```diff
@@ -0,0 +1,423 @@
...
+* Marker - String - Indicates the result of the Winsorisation method for
+each target value
+  * **W** - Winsorised. Marked against any value that could be Winsorised
+  * **NW_FE** - Not Winsorised Fully Enumerated. Marked when design weight is
+  1 for expansion estimation.
+  * **NW_AG** - Not Winsorised a*g. Marked
+  in ratio estimation when the product of
+  design weight and calibration factor is <= 1.
+
+The methodological specification avaialble at [https://statisticalmethodslibrary.ons.gov.uk/](https://statisticalmethodslibrary.ons.gov.uk/) contains a detailed breakdown of the mathematics used in Winsorisation.
```
Comment (jhmjam) [inline method-info/winsorisation/winsorisation.md:52]: * available

may use the word 'here' as a link label

File: method-info/winsorisation/winsorisation.md [hunk 4]
```diff
@@ -0,0 +1,423 @@
...
+
+When _target value_, **$y$** is <= _threshold_, **k** the _modified target_ equals the _target value_ :
+
+$$ y^* = y $$
+
+When _target value_, **$y$** is > _threshold_, **k** the _modified target_ is calculated as :
+
+$$ y^* = \frac{y}{w} +(1-(\frac{1}{w}))*k $$
+
+The _outlier weight_, **$o$** can then be calculated as:
```
Comment (jhmjam) [inline method-info/winsorisation/winsorisation.md:128]: should it be **o** or should the example above be like this one?

**Review by jhmjam — CHANGES_REQUESTED** [review]

File: method-info/winsorisation/example-data/winsorisation_expansion_input.csv (hunk 1 above)
Comment (gibbardsteve) [reply method-info/winsorisation/example-data/winsorisation_expansion_input.csv:2]: Partly from the original method spec and partly from the testing alongside the method.  This is being reviewed by methodology.

File: method-info/winsorisation/winsorisation.md (hunk 2 above)
Comment (gibbardsteve) [reply method-info/winsorisation/winsorisation.md:11]: Updated - thanks.

File: method-info/winsorisation/winsorisation.md (hunk 3 above)
Comment (gibbardsteve) [reply method-info/winsorisation/winsorisation.md:52]: Changed the typo on the available. Not sure if you had a comment about the link, I have purposefully left it pointing to the URL of the website to make it clear there is a supporting website.

File: method-info/winsorisation/winsorisation.md (hunk 4 above)
Comment (gibbardsteve) [reply method-info/winsorisation/winsorisation.md:128]: I am not sure what your comment means, the calculation of **o** is essentially the modified target value divided by the original value, if there isn't a modification to the target value then it becomes the original value divided by the original value which equates to 1. The algebra is ok as far as I can see.

File: method-info/winsorisation/winsorisation.md [hunk 5]
```diff
@@ -0,0 +1,423 @@
...
+
+```
+
+## Example Data
+
+The scenarios outlined above are captured in the csv files under the example-data directory where you found this user guide.
+
+## Additional Information
+
+The ONS Statistical Methods Library at [https://statisticalmethodslibrary.ons.gov.uk/](https://statisticalmethodslibrary.ons.gov.uk/) contains further information about the methods including:
```
[the remark this answers is not in the document]
Comment (gibbardsteve) [reply method-info/winsorisation/winsorisation.md:414]: I think I'd prefer for it just to point to the direct URL for the website to make it clear there is a supporting website dedicated to info about the methods. This is consistent with the Thousand Pounds user notes which have previously been approved.

**Review by jhmjam — DISMISSED** [review]

**Review by jhmjam — APPROVED** [review]

**Review by JasonBellONS — APPROVED** [review]

**Pull request merged**